certbot-nginx/local-oldest-requirements.txt

```diff
@@ -1,2 +1,2 @@
 acme[dev]==0.26.0
-certbot[dev]==0.22.0
+certbot[dev]==0.31.0
```

Comment:
```suggestion
-e .[dev]
```

Reply:
Hi @ohemorange Tried this but it still fails. You mentioned apache, should the changes be made there also or instead? Feel free to mod this PR, I have no clue of the testing deps of certbot.

Reply:
Yes, you're right, Apache needs the change as well. I can't push to your branch, the change is the same as in `certbot-nginx/setup.py`.